decidim-assemblies/lib/decidim/assemblies/admin_engine.rb

```diff
@@ -16,6 +16,7 @@ class AdminEngine < ::Rails::Engine
 
       routes do
         resources :assemblies_types
+        resource :assemblies_settings, only: [:edit, :update], controller: "assemblies_settings"
```

Comment:
```suggestion
        resource :assembly_settings, only: [:edit, :update], controller: "assemblies_settings"
```

Just to justify why it should be called different from `assemblies_types`: `assemblies_types` affect to `all` the assemblies while `assembly_settings` is for an specific assembly only.

Reply:
I figured when we set the settings it will be applied to all the "assemblies" of the organisation. Hence, I went with the plural semantics everywhere.

Reply:
But it is not right? lets double check this before doing extra work!

Reply:
I understand that is a global setting for all assemblies of the organization, that's why is in plural ("assemblies")
So if the chart setting is disabled, none of the assemblies' homepage will render the chart.

Reply:
Exactly, these are general settings for the whole `decidim-assemblies` module.

Reply:
Thank you all, sorry for the confusion then.

Reply:
👍 no worries, thank you too